pkgs/development/tools/build-managers/scons/default.nix

```diff
@@ -5,10 +5,7 @@ let
     python = python3;
   };
 in {
-  scons_3_0_1 = (mkScons {
-    version = "3.0.1";
-    sha256 = "0wzid419mlwqw9llrg8gsx4nkzhqy16m4m40r0xnh6cwscw5wir4";
-  }).override { python = python2; };
+  scons_3_0_1 = throw "scons 3.0.1 was removed from nixpkgs since it only supports python2";
```

Comment:
This was not exposed in `all-packages.nix` so maybe I will just remove it instead of throwing.